SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -15,19 +15,72 @@ \chapter{Configuring and Building ITK}
 
 ITK has been developed and tested across different combinations of operating
 systems, compilers, and hardware platforms including Microsoft Windows, Linux on
-various architectures, Solaris/UNIX, macOS, and Cygwin. Kitware is committed
-to support the following compilers for building ITK:
+various architectures, UNIX, macOS, and Cygwin. Dedicated community members and
+Kitware are committed to providing long-term support of the most prevalent
+development environments (Visual Studio, macOS, and Linux) for building ITK:
+
+Compiler variants will be supported for the duration that the associated operating
+system vendors commit to in their long-term stable platforms. For example the gcc
+compilers supported will mirror the compiler support in the RedHat lifecycle, the
+apple clang compilers will mirror the support lifecycle of the compiler by Apple, and
+the Visual Studio series will have approximately a 10 year support lifecycle.
 
 % List updated according to information provided here:
 % https://www.itk.org/Wiki/ITK_Release_4/Modern_C++#Fully_Committed_to_Support
 
+For example as of 2017 the following time schedule is expected for supporting
+these compiler environments.
+
+%%
+%% The following compilers should have nightly dashboard in the "Expected builds"
+%% to ensure that support is maintained.
+%%
 \begin{itemize}
-\item GCC 4.x
-\item Visual Studio 9 (until 2018), 10 (until 2020), 11 (until 2022), 12 (until 2023), 13 (until 2025)
-\item Intel Compiler Suite 11.x, 12.x (including macOS release)
-\item Win32-mingw-gcc-4.5
-\item Apple clang-600.0.56
-\item Clang 3.3 and later
+\item GCC
+  \begin{itemize}
+      \item 4.4 (From 2009-until 2018)  %% April 21, 2009 -- build_name "Linux-x86_64-gcc4.4"
+      \item 4.8.2 (From 2015-until 2020)  %% June 26, 2015  -- build_name "Ubuntu-g++-4.8.2-Release"
+      %\item 5.x %% Release date April 22, 2015
+      %\item 6.x %% April 27, 2016
+      %\item 7.x %% May 2, 2017
+   \end{itemize}
+%% I am concerned about this policy.  Our support dates extend well beyond what even MS is willing to support.
+%% We are promising to support ITK building under these compilers even after MS will support the compiler itself.
+\item Visual Studio % https://en.wikipedia.org/wiki/Microsoft_Visual_Studio
```

Comment:
Then we should remove or modify the sentence
`and the Visual Studio series will have approximately a 10 year support lifecycle.` 
on L26, right?

Reply:
Done.